### ISSUE TITLE
Fetch from Mintbase / Paras with context methods

### ISSUE THREAD

**Issue by gabehamilton** [near/queryapi#440]
**Is your feature request related to a problem? Please describe.**
For NFT based groups and other features that use NFTs, we need a table of all NFTs.

**Describe the solution you'd like**
Rather than build our own NFT indexer, add context methods to query the Mintbase and Paras apis.

**Comment by pkudinov** [comment]
I don’t think we can introduce any specific external indexer into a context object (like context.nft) – this is too specific. The indexer code is stored on-chain, which means whoever else runs QueryAPI might need to support this.
I see 2 options:

1. Figure out how we can introduce external GraphQL schemas to QueryAPI Hasura. This config should be stored at the indexer registry and applied at provisioning stage. This way context.graphql will work
2. Implement a generic context.httpFetch() method for any web calls
